spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -167,45 +167,65 @@
     end
 
     describe 'roles' do
-      before do
-        get :show, params: { id: transcription.id }
+      shared_examples 'lockable transcription' do
+        let(:locked_transcription) { create(:transcription, locked_by: 'kar-aniyuki', lock_timeout: (DateTime.now + 1.hours)) }
+
+        it 'locks the transcription if not already locked' do
+          get :show, params: { id: transcription.id }
+          expect(json_data['attributes']['locked_by']).to eq(user.login)
+        end
+
+        it 'does not lock the transcription if it is already locked' do
+          get :show, params: { id: locked_transcription.id }
+          expect(json_data['attributes']['locked_by']).to eq(locked_transcription.locked_by)
```

Comment:
The assumption made here is that `locked_transcription.locked_by` will not have changed because it was already set. But this line is also based on the assumption that `locked_transcription` isn't refreshed from the db at any point. If it were, then this is a "the thing i was trying to change has changed" test rather than the inverse. I suggest using the actual string in the expect rather than the instance attr to be clear and certain. 

Also, you can add an additional `expect(locked_transcription).not_to receive(:lock!)` before the `get` to make sure that it's not calling that method at all, in case calling it is non-transformative in the spec.

Reply:
Cam's `expect().not_to change()` suggestion would also solve this for me, your call

Reply:
Definitely. good thing to keep in mind. on it.